climada/util/alpha_shape.py

```diff
@@ -0,0 +1,99 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Compute the concave hull of a set of points.
+"""
+
+import numpy as np
+
+from shapely.ops import cascaded_union, polygonize
+from scipy.spatial import Delaunay
+import math
+import shapely.geometry as geometry
+import pylab as pl
+
+def alpha_shape(points, alpha):
+    """
+    Compute the alpha shape (concave hull) of a set
+    of points.
+    Source: https://thehumangeo.wordpress.com/2014/05/12/drawing-boundaries-in-python/
+    @param points: Iterable container of points.
+    @param alpha: alpha value to influence the
+        gooeyness of the border. Smaller numbers
+        don't fall inward as much as larger numbers.
+        Too large, and you lose everything!

```

Comment:
?

Reply:
I'll update when we decide to keep it =)